packages/node/src/plugins/loadScript.ts

```diff
@@ -58,17 +52,34 @@ export const executeLoadTemplate = `
     if(!name) {
       throw new Error('__webpack_require__.l name is required for ' + url);
     }
+    console.log('executeLoad: loading chunk', {
+      url,callback,name
+    });
     if (typeof global.__remote_scope__[name] !== 'undefined') return callback(global.__remote_scope__[name]);
+
     const vm = require('vm');
     (global.webpackChunkLoad || global.fetch || require("node-fetch"))(url).then(function (res) {
       return res.text();
     }).then(function (scriptContent) {
+      console.log('in script loading promise for', name);
       try {
         const vmContext = {exports, require, module, global, __filename, __dirname, URL,console,process,Buffer, ...global, remoteEntryName: name};
         const remote = vm.runInNewContext(scriptContent + '\\nmodule.exports', vmContext, {filename: 'node-federation-loader-' + name + '.vm'});
-        global.__remote_scope__[name] = remote[name] || remote;
-        global.__remote_scope__._config[name] = url;
-        callback(global.__remote_scope__[name])
+        const foundContainer = remote[name] || remote
+        if(!global.__remote_scope__[name]) {
```

Comment:
moving container proxying up to here so that remotes appear in memory under same address when registered on host.